app/app.go

```diff
@@ -31,6 +39,7 @@ type Config struct {
 }
 
 func buildScaledContext(ctx context.Context, kubeConfig rest.Config, cfg *Config) (*config.ScaledContext, *clustermanager.Manager, error) {
+	NewDaemon()
```

Comment:
What is the motivation for doing thi sin the buildScaledContext function? It does not seem like the right place for this. Woudl make more sense in the function that calls buildScaledContext